app/main.js

```diff
@@ -471,8 +472,11 @@ function startMicrobreak () {
 
     microbreakWinLocal.loadURL(modalPath)
     microbreakWinLocal.setVisibleOnAllWorkspaces(true)
-    microbreakWinLocal.setAlwaysOnTop(true, 'screen-saver')
+    microbreakWinLocal.setAlwaysOnTop(actAsRegularWindow, 'screen-saver')
     if (microbreakWinLocal) {
+      microbreakWinLocal.on('close', (e) => {
```

Comment:
What is this doing?

Reply:
This prevents the break window from being closed.

Reply:
because:
- allowing the window to be `focusable` lets the users ALT-F4, CTRL-W to close it
- adding the window to the taskbar lets the users close the window by right-clicking it in the taskbar
- adding the titlebar lets the users press the cross on the window to close it

AFAIK there is no way to remove those close buttons in electron besides removing the frame (which I've avoided to let users drag and minimize the window in this mode)

Reply:
Ah, ok :) Makes sense